### PR TITLE
Do not merge: WIP

### PR DESCRIPTION
**Pull request by Tulean**
---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4147/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5768/ : FAILURE in 14m 18s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4157/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5780/ : FAILURE in 14m 13s

File: libs/sdk-model/src/execution/executionDefinition/factory.ts [hunk 1]
```diff
@@ -158,6 +160,24 @@ export function defWithSorting(definition: IExecutionDefinition, sorts: ISortIte
     return defSetSorts(definition, sorts);
 }
 
+/**
+ * Changes additional execution configuration in the definition.
+ * Any additional execution configuration settings accumulated so far WILL be wiped out.
+ *
+ * This function MUST be used to implement IPreparedExecution.withExecConfig();
+ *
+ * @param definition - definition to alter with sorting
```
Comment (david-kubecka) [inline libs/sdk-model/src/execution/executionDefinition/factory.ts:169]: Mistaken mention of sorting - probably copy paste...

File: libs/sdk-model/src/execution/executionDefinition/index.ts [hunk 2]
```diff
@@ -29,6 +29,18 @@ export interface IPostProcessing {
...
 }
 
+/**
+ * Contains any configiration that should be part of execution
+ *
+ * @public
+ */
+export interface IExecutionConfig {
+    /**
+     * Data sampling is only available for tiger with Vertica
```
Comment (david-kubecka) [inline libs/sdk-model/src/execution/executionDefinition/index.ts:39]: replace `for tiger with Vertica` with `in Tiger for specific databases` (we will support sampling for more DBs in the future)

**Pull request closed without merging**